client/my-sites/plugins/controller.js

```diff
@@ -186,16 +186,18 @@ function renderProvisionPlugins( context ) {
 }
 
 const controller = {
-	plugins( filter, context, next ) {
+	plugins( context, next ) {
+		context.params.pluginFilter = context.params.pluginFilter || 'all';
```

Comment:
☝️  `'all'` is here because of [`pluginsController.plugins.bind( null, 'all' )`](https://github.com/Automattic/wp-calypso/pull/20019/files#diff-540b4f9d3b87d29d4223bdefe9f97b94L71).

Reply:
Hmm, we tend to try and avoid re-assignments like this, especially when it comes to mutating objects (and when it can be done more elegantly 😛). What about

```js
const { pluginFilter: filter = 'all' } = context.params;
```

?

Reply:
⚡️ Totally! :-)